crates/bevy_render/src/camera/camera.rs

```diff
@@ -555,6 +555,8 @@ pub fn camera_system<T: CameraProjection + Component>(
                 }
             }
         }
+
+        camera.computed.old_viewport_size = viewport_size;
```

Comment:
Probably should surround this with a inequality check to ensure that cameras aren't always marked as changed.